in_topic: "/usb_cam/image_raw"
result_topic: "ros_opencl_tests_cpu/result"
blur_radius: 50